logs/parse.go

```diff
@@ -72,6 +73,7 @@ var LogPrefixCustom8Regexp = regexp.MustCompile(`(?s)^\[` + PidRegexp + `\]: \[`
 var LogPrefixCustom9Regexp = regexp.MustCompile(`(?s)^` + TimeRegexp + ` ` + HostAndPortRegexp + ` ` + UserRegexp + ` ` + AppRegexp + ` \[` + SessionIdRegexp + `\] \[` + PidRegexp + `\] ` + LevelAndContentRegexp)
 var LogPrefixCustom10Regexp = regexp.MustCompile(`(?s)^` + TimeRegexp + ` \[` + PidRegexp + `\]: \[` + LogLineCounterRegexp + `-1\] db=` + DbRegexp + `,user=` + UserRegexp + ` ` + LevelAndContentRegexp)
 var LogPrefixCustom11Regexp = regexp.MustCompile(`(?s)^pid=` + PidRegexp + `,user=` + UserRegexp + `,db=` + DbRegexp + `,app=` + AppInsideBracketsRegexp + `,client=` + HostRegexp + ` ` + LevelAndContentRegexp)
+var LogPrefixCustom12Regexp = regexp.MustCompile(`(?s)^user=` + UserRegexp + `,db=` + DbRegexp + `,app=` + AppInsideBracketsRegexp + `,client=` + HostRegexp + ` ` + LevelAndContentRegexp)
```

Comment:
I noticed I missed this in `Custom11` before, but why `AppInsideBracketsRegexp` instead of `AppRegexp`? I see that `Custom6` is also using `AppInsideBracketsRegexp` ostensibly outside of brackets, but `Custom9` is using just `AppRegexp`. Are these misnamed or am I missing something?

Reply:
Oh I see, based on the test, it looks like the value itself has brackets around it? But are the brackets not just part of the application name in that case? And what if they are not set by the application. `%a` itself does *not* add brackets, right?

Reply:
Nice, that is a good catch, I certainly got this wrong :)

Adjusted to use the correct regexp for the new prefixes, and also adjusted a few others that were incorrect in this regard.